IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedBldgElements/Entities/IfcFillElementType/DocEntity.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcFillElementType" UniqueId="8ed7f4d1-b183-4e38-b84e-6cc0271430dc" BaseDefinition="IfcBuiltElementType" EntityFlags="32">
+	<Attributes>
+		<DocAttribute Name="PredefinedType" UniqueId="60b6de99-fd5c-44c5-aeb6-2ecac4250306" DefinedType="IfcFillElementTypeEnum" AttributeFlags="1" />
+	</Attributes>
+	<WhereRules>
+		<DocWhereRule Name="CorrectPredefinedType" UniqueId="d95cedbb-a455-42d7-8e64-71c6523087bb">
```

Comment:
Please check this rule. I have the impression that it does not make sense for IfcFillElement*Type*.

Reply:
As far as I can see, the rule is defined the same way as for other type objects. @SergejMuhic: can you please check also before merging?

Reply:
The rule does not make sense indeed. The fix is simple though. `PredefinedType` on `IfcTypeObject` children should be mandatory. No optional. 😄

Reply:
Oh, missed that mistake. Thank you!
Fixing now. Also add documentation for PredefinedType that is obviously missing as well :-)

Reply:
Done.